zerver/actions/streams.py

```diff
@@ -1273,11 +1273,11 @@ def do_change_stream_permission(
             stream.id, include_deactivated_users=False
         ).values_list("user_profile_id", flat=True)
 
-        old_can_access_stream_user_ids = set(stream_subscriber_user_ids) | {
+        old_can_access_stream_metadata_user_ids = set(stream_subscriber_user_ids) | {
```

Comment:
Should this also contain users in `can_administer_channel_group` and `can_add_subscribers_group`?

Reply:
Ohh I see, this is fixed in further commits. Any reason for a separate commit?

Reply:
I think probably just sanity; it's a very big stack of commits.